### PR TITLE
MSPSDS-789: Add sentry integration

### PR DESCRIPTION
**Pull request by broder**

### CODE REVIEW

File: web/config/application.rb [hunk 1]
```diff
@@ -26,8 +26,19 @@ class Application < Rails::Application
     # This changes Rails timezone, but keeps ActiveRecord in UTC
     config.time_zone = "Europe/London"
 
+    config.rails_activesupport_breadcrumbs = true
+
+    # https://github.com/getsentry/raven-ruby/issues/494
```
Comment (jasiekmiko) [inline web/config/application.rb]: Maybe just being slow, but took me a while to understand how this comment is relating to the code.
Perhaps pointing directly to the solution with `https://github.com/getsentry/raven-ruby/issues/494#issuecomment-417277866`?

File: web/config/application.rb [hunk 2]
```diff
@@ -26,8 +26,19 @@ class Application < Rails::Application
...
     config.exceptions_app = self.routes
 
+    # Inject Sentry logger breadcrumbs
+    require 'raven/breadcrumbs/logger'
+
+    Raven.configure do |config|
+      config.dsn = ENV['SENTRY_DSN']
+      config.sanitize_fields = Rails.application.config.filter_parameters.map(&:to_s)
```
Comment (jasiekmiko) [inline web/config/application.rb]: Does this essentially say "we already have a list of parameters to be excluded from logging out, use it for raven, too"?

File: web/config/application.rb (hunk 1 above)
Comment (broder) [reply web/config/application.rb]: The comment came from https://github.com/getsentry/raven-ruby/blob/master/examples/rails-5.0/config/application.rb#L15 - I think I'll update the comment to point to that instead.

File: web/config/application.rb (hunk 2 above)
Comment (broder) [reply web/config/application.rb]: Yep

**Pull request merged**